src/openvpn/misc.h

```diff
@@ -175,6 +175,15 @@ void output_peer_info_env(struct env_set *es, const char *peer_info);
 
 #endif /* P2MP_SERVER */
 
+
+#define random_hex(buf, _sz) { \
+        int sz = _sz; \
+        srand(time(NULL)); \
+        while (sz-- > 0){ \
+                buf[sz] = rand()%256; \
+        } \
+}
```

Comment:
Please no custom random function. This seems to just generate _sz number of bytes. We already have `rand_bytes`